tests/test_datasets/test_transforms/test_transforms.py

```diff
@@ -1,4 +1,5 @@
 # Copyright (c) OpenMMLab. All rights reserved.
+# yapf:disable
```

Comment:
Is it necessary?

Reply:
Without this, isort and yapf would conflict.

Reply:
> Is it necessary?
Without this, isort and yapf would conflict.